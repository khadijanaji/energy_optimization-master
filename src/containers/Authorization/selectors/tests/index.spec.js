import appSelector from "../";

const stateUserConnected = {"login": {"infoUser": {"token": "FakeToken"}}};
const stateUserNotConnected = {"login": {"infoUser": {"token": ""}}};

describe("Selector", () => {
  describe("App selector", () => {
    it("User is connected", () => {
      expect(appSelector(stateUserConnected)).toMatchSnapshot();
    });

    it("User is not connected", () => {
      expect(appSelector(stateUserNotConnected)).toMatchSnapshot();
    });
  });
});
