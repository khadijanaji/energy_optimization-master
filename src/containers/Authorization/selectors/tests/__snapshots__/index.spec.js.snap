// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Selector App selector User is connected 1`] = `
Object {
  "isUserConnected": true,
  "user": Object {
    "token": "FakeToken",
  },
}
`;

exports[`Selector App selector User is not connected 1`] = `
Object {
  "isUserConnected": false,
  "user": Object {
    "token": "",
  },
}
`;
